CONTRIBUTING.md

```diff
@@ -193,11 +193,11 @@ As languages differ quite a lot, here is a set of captures available to you when
 @comment               ; line and block comments
 @comment.documentation ; comments documenting code
 
-@comment.error         ; error-type comments (e.g., `DEPRECATED:`)
-@comment.warning       ; warning-type comments (e.g., `WARNING:`, `FIX:`)
-@comment.hint          ; note-type comments (e.g., `NOTE:`)
+@comment.error         ; error-type comments (e.g. `DEPRECATED:`)
+@comment.warning       ; warning-type comments (e.g. `WARNING:`, `FIX:`)
+@comment.hint          ; note-type comments (e.g. `NOTE:`)
 @comment.info          ; info-type comments
```

Comment:
```suggestion
@comment.info          ; info-type comments (e.g. `INFO:`)
```

Reply:
I think my thinking was "no need to repeat the actual capture name", but it can't hurt to be explicit.

Reply:
Or wait for the comment PR to land, then we can adjust?

Reply:
Yeah, I think I want to wait for that. This deserves some thinking about.